generation_config.yaml

```diff
@@ -1,8 +1,18 @@
 googleapis_commitish: 3cef095370b3176f67542e527ffde81e019df091
-# the libraries are ordered with respect to library name, which is
-# java-{library.library_name} or java-{library.api-shortname} when
-# library.library_name is not defined.
+# the libraries are ordered with respect to library name, which defaults to
+# {library.folder_name} or java-{library.library_name} or
+# java-{library.api-shortname} in sequence (i.e. if first is None, then try second and so on).
 libraries:
+- api_shortname: showcase
+  folder_name: showcase # prevents java-showcase
```

Comment:
I feel it is probably too much effort to introduce a new argument `folder_name`, just to prevent `showcase` to be renamed to `java-showcase`. I think it is OK to rename it.

Reply:
I removed this new flag